src/app/components/NonScrollingRouterLink/index.tsx

```diff
@@ -0,0 +1,6 @@
+import React, { FC } from 'react'
+import { Link } from 'react-router-dom'
+
+export const NonScrollingRouterLink: FC<Parameters<typeof Link>[0]> = React.forwardRef((props, ref) => (
```

Comment:
You can make it cleaner by using utility type `FC<ComponentPropsWithRef<typeof Link>>`.

Reply:
Thanks for the tip, done.

Reply:
Do we have further plans for this abstraction, or is it just a longer way of adding "preventScrollReset"?

Reply:
As far as I am concerned, it serves a dual purpose:
 - Avoid having to spell out `preventScrollReset={true}`
 - Oftentimes we want to import ReactRouter's `Link` as `RouterLink`, since we also want to work with Mui's `Link` in the same namespace. This is a bit tedious, and requires manual editing (no automati imports.) However, using `NoScrollingRouterLink` avoids this problem. (Along the same lines, I think it would also make sense to create an alias to React Router's `Link` called `RouterLink`. It would make it easier to import.)